src/app/config/config.c

```diff
@@ -2459,6 +2459,8 @@ static const struct {
   takes_argument_t takes_argument;
   /** If not CMD_RUN_TOR, what should Tor do when it starts? */
   tor_cmdline_mode_t command;
+  /** If nonzero, set the quiet level to this. 1 is "hush", 2 is "quiet" */
```

Comment:
Can we please have this as a define or an enum?
I can never remember which is quieter :-)